raiden/transfer/channel.py

```diff
@@ -1671,11 +1674,18 @@ def handle_receive_lock_expired_light(
             payment_id,
             1,
             state_change.lock_expired,
-            True,
+            False,
             LightClientProtocolMessageType.PaymentExpired,
             state_change.lock_expired.recipient
         )
-        events = [store_lock_expired]
+
+        send_processed = SendProcessed(
```

Comment:
This isnt correct. 

Here we are on the situation that a HUB receives a lock expired msg that its recipient is a handled light client right? 

The ACK messages must be signed by the LC, not by the hub. Think about the SendProcessed handling 

```
    @staticmethod
    def handle_send_processed(raiden: "RaidenService", processed_event: SendProcessed):
        processed_message = message_from_sendevent(processed_event)
        raiden.sign(processed_message) # HERE THE HUB SIGNS THE MESSAGE
        raiden.transport.hub_transport.send_async(processed_event.queue_identifier, processed_message)
```

Basically the hub sends the ACK signing the message itself.

If this works is basically because the processed has the same id of the lock expired and it removes the L.E from the retry queue, but the protocol must be done between 2 participants that signs the messages, a 3rd one (the HUB on this case) must not sign any offchain msg.

This makes me think, we have already implemented the reception of L.E for light clients (basically the LC receives a L.E from the long polling and sends a delivered and processed message), so, i dont know why this was needed

Reply:
I mean, if the ACK were missing, i dont know how p2p payments works after a lock expired...

AFAIK here we dont create/dispatch SendProcessed or SendDelivered cause the LC sdk has the logic or responding with those messages when it receives a L.E

Reply:
I can remove that if you want but not processed message will be out, the LC dont send that out, i checked that, i didn't knew that the processed message should be signed by the LC too, anyway is not being sent out at all so i don't know what to tell you here. I can remove it anyway

Reply:
Lets talk about this on the daily

Reply:
> Lets talk about this on the daily

can you guys please briefly document the resolution on this, if there is one?

Reply:
Basically we discussed why isnt correct and we must check why the LC was not responding to that messages.